src/lessons/03-Props-vs-State/exercise.js

```diff
@@ -21,55 +21,66 @@ import PropTypes from "prop-types";
 import * as styles from "./styles";
 import data from "./data";
 
-class Tabs extends React.Component {
-  static propTypes = {
```

Comment:
Please add `propTypes` to component. They are runnable docs. You will thank yourself for doing this later on